src/browser/components/SkillItem.react.js

```diff
@@ -16,46 +15,23 @@ export default class SkillItem extends Component {
   handleChange(level, e) {
     const {actions: {saveSkill}} = this.props;
     saveSkill(this.props.data, level);
-    this.props.resetErrorStatus();
   }
 
-  setHintVisibility(visible) {
-    this.setState({hintDisplayed: visible});
-  }
-
-  handleMouseEnter = () => this.setHintVisibility(true);
-  handleMouseLeave = () => this.setHintVisibility(false);
-
   render() {
-    let hint;
-    const cb = {
-      onMouseEnter: this.handleMouseEnter,
-      onMouseLeave: this.handleMouseLeave
-    };
-
-    if (this.state.hintDisplayed) {
-      hint = (<div {...cb} className="hint">{this.props.data.name}</div>);
+    const starsLabels = [];
+    for (let i = 5; i >= 1; i--) {
```

Comment:
How 5 can be const if we change it in loop? I checked it and it show error on i-- place. And loop is decrementing because this elements inside was before in that order. It change "stars" classes names order and i'm not sure if it also change final JSON result.